src/Bridge/Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -347,6 +377,8 @@ protected function createWrapCase(bool $caseSensitive): \Closure
      *
      * @param string $value
      *
+     * @deprecated
```

Comment:
Can you add a message explaining what the user should use instead? A call to `trigger_error` should also be added.